pom.xml

```diff
@@ -62,7 +62,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.build.timestamp.format>yyyyMMddhhmm</maven.build.timestamp.format>
-    <javacpp.version>${project.version}</javacpp.version>
+    <javacpp.version>1.1</javacpp.version>
```

Comment:
I do not know about the target environment but with this it would be much easier to deploy (it just gets packages from Maven Central)